include/xrpl/basics/base_uint.h

```diff
@@ -632,6 +632,18 @@ to_string(base_uint<Bits, Tag> const& a)
     return strHex(a.cbegin(), a.cend());
 }
 
+template <std::size_t Bits, class Tag>
+inline std::string
+to_short_string(base_uint<Bits, Tag> const& a)
+{
+    // LCOV_EXCL_START
+    if constexpr (base_uint<Bits, Tag>::bytes <= 4)
+        return to_string(a);
+    else
+        // LCOV_EXCL_STOP
```

Comment:
`if constexpr` confusing code coverage 🥇

Reply:
> `if constexpr` confusing code coverage 🥇

Changed this to just a `static_assert`, since we already have `uint32_t`, etc.